shell/initialize/entry-helpers.js

```diff
@@ -264,12 +264,8 @@ export async function mountApp(appPartials, VueClass) {
 
   // Add beforeEach router hooks
   router.beforeEach(render.bind(vueApp));
-  router.beforeEach((from, to, next) => {
-    if (from?.name !== to?.name) {
-      updatePageTitle(getVendor());
-    }
-
-    next();
+  router.afterEach((from, to) => {
```

Comment:
Relevant note in the router docs.

![image](https://github.com/rancher/dashboard/assets/55104481/d3550b5c-5e77-4cff-a968-b1f6eb406eb0)